packages/next/src/build/swc/index.ts

```diff
@@ -8,6 +8,7 @@ import { getParserOptions } from './options'
 import { eventSwcLoadFailure } from '../../telemetry/events/swc-load-failure'
 import { patchIncorrectLockfile } from '../../lib/patch-incorrect-lockfile'
 import { downloadWasmSwc } from '../../lib/download-wasm-swc'
+import { spawn } from 'child_process'
```

Comment:
Why did we switch to spawn from `cross-spawn`? Also would be nice to keep the require to when it's actually needed.

Reply:
I was wondering if https://vercel.slack.com/archives/C04KC8A53T7/p1676439048600469 issue is related with non-native spawn, jest hangs on somewhere and cause sigterm for some tests. I don't honestly have concrete proof, more of trial-and-error to see if it makes a difference or not though.